sssd_test_framework/utils/authentication.py

```diff
@@ -152,6 +152,16 @@ def test_example(client: Client, ldap: LDAP, kdc: KDC):
         """
         return KerberosAuthenticationUtils(self.host, ssh)
 
+    def passwd(self, ssh: SSHClient) -> PasswdUtils:
+        """Return object for password management connected to given ssh client.
+
+        :param ssh: SSH connection for the target user.
+        :type ssh: SSHClient
+        :return: PasswdUtils object.
+        :rtype: PasswdUtils
+        """
+        return PasswdUtils(self.host, ssh)
```

Comment:
It is not needed to put ssh there, we can use the root ssh connection from self.host.ssh to make it simpler.

You just need to spawn `su - {username} -c passwd`